examples/chat-rn-expo-clerk/app/_layout.tsx

```diff
@@ -33,6 +44,17 @@ export default function RootLayout() {
     );
   }
 
+  useEffect(() => {
+    if (fontsLoaded) {
+    } else {
+      SplashScreen.preventAutoHideAsync();
+    }
+  }, [fontsLoaded]);
+
+  if (!fontsLoaded) {
+    return null;
+  }
```

Comment:
Picked these changes from #1848 as before the user wasn't redirected to the auth screen on logout.